test/controllers/eventController.spec.js

```diff
@@ -0,0 +1,420 @@
+let mocha = require('mocha')
```

Comment:
So, most of the data is already seeded for you, in terms of users. So, instead of creating new user records at the beginning, just load them. See the `rsoController.spec.js` to see how to do it.